docs/serving/autoscaling/container-freezer.md

```diff
@@ -0,0 +1,54 @@
+# Configuring container-freezer
+
+When container-freezer is enabled, queue-proxy calls an endpoint API when its traffic drops to zero or scales up from zero.
+
+Within the official endpoint API implementation container-freezer, the running process is frozen when the pod's traffic drops to zero, and resumed when the pod's traffic scales up from zero. The endpoint API is a user-specific implementation. For example, you can implement it as a billing component because it knows when the requests are being handled.
```

Comment:
Also, and this is a nit, I wonder if "official" is the right word here... maybe "community-maintained" or something similar?

Reply:
community-maintained is a good option

Reply:
fixed